cabal-edit.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version:      3.0
+cabal-version:      2.4
```

Comment:
Cabal version `3.0` works with a somewhat recent commit from `nixpkgs-unstable` from the `nixpkgs-channels` repo (which has `ghc-8.8.3` as the default compiler version), might be nice to use that instead